ci_framework/roles/libvirt_manager/README.md

```diff
@@ -27,3 +27,4 @@ Used for checking if:
 * `cifmw_libvirt_manager_compute_disksize`: (Integer) The size of the disk in GB per compute. Defaults to `20`.
 * `cifmw_libvirt_manager_compute_cpus`: (Integer) The amount of vCPUs per compute. Defaults to `1`.
 * `cifmw_libvirt_manager_apply_virtproxy_patch` (Boolean) Apply patch virtproxy patch for improved libvirt stability. This is set to `true` by default.
+* `cifmw_libvirt_manager_net_prefix_add` (Boolean) Adds `cifmw` prefix to the network resources managed by this role. By default, it is set to `true`.
```

Comment:
`cifmw-` prefix :)

Reply:
Done... will be fixed in the next commit.